modules/tfm/tfm/boards/common/crypto_keys.c

```diff
@@ -9,64 +9,223 @@
 
 #include "psa/crypto_types.h"
 #include "tfm_plat_crypto_keys.h"
+#include "tfm_builtin_key_ids.h"
+#include "tfm_builtin_key_loader.h"
+#include "psa_manifest/pid.h"
+#include "tfm_spm_log.h"
+
 #include <hw_unique_key.h>
 #include <nrf_cc3xx_platform_kmu.h>
 #include <nrf_cc3xx_platform_identity_key.h>
 #include <nrf_cc3xx_platform_defines.h>
 
+#define TFM_NS_PARTITION_ID -1
```

Comment:
```suggestion
#define TFM_NS_PARTITION_ID (-1)
```

IIRC you can get a precedence issue from -